build.gradle

```diff
@@ -20,6 +20,10 @@ checkstyle {
     toolVersion = '10.2'
 }
 
+checkstyleMain {
+    exclude("**/org/openapitools/client/**")
+}
+
```

Comment:
Excluded generated `openapi` code from `checkstyle`. But we probably want to clean it up as we develop.